home-data/src/main/resources/application.properties

```diff
@@ -1,5 +1,6 @@
-spring.datasource.url=${DATASOURCE_URL:jdbc:postgresql://localhost:5433/postgres}
+spring.datasource.url=${DATASOURCE_URL:jdbc:postgresql://localhost:5432/postgres}
```

Comment:
I think the port here should correlate to the port we are using in docker-compose and in maven liquibase plugin. So we have consistency for local development.

Reply:
You're right. Corrected